### PR TITLE
managedsave: add expected vcpus for cpu_mode test

### PR DESCRIPTION
**Pull request by crazyAxe**
the amount of vcpus should be equal or more than (sockets * cores * threads)

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py [hunk 1]
```diff
@@ -289,6 +289,7 @@ def build_vm_xml(vm_name, **dargs):
                 if cpu_topology:
                     cpuxml.topology = cpu_topology
                 vmxml.cpu = cpuxml
+                vmxml["vcpu"] = int(params.get("vcpus_nums"))
```
Comment (rbian) [inline libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py]: Why not use vmxml.vcpu to set the value for it to keep consistent with other code?

Comment (crazyAxe) [inline libvirt/tests/src/virsh_cmd/domain/virsh_managedsave.py]: 'vmxml.vcpu'  is indeed better. Actually i just found the vmxml["vcpu"] in function set_vm_vcpus to set the value ,  so used it here. I'll  improve it, thank.

**Comment by waynesun09** [comment]
@crazyAxe You need clean up this PR, better solution is use git branches, keep your fix in develop branch rather than master. Then push from devel branch and request merge.
Also you could consider add your sign-off email address associated to your github accound, then github could identify your commit.

**Comment by waynesun09** [comment]
And you could take a look of the doc:
http://autotest.readthedocs.io/en/latest/main/developer/SubmissionChecklist.html

**Comment by waynesun09** [comment]
@crazyAxe paste the wrong doc link in last comment, update it to:
https://www.redhat.com/archives/virt-test-devel/2013-April/msg00190.html

the mail is quite a long time ago, and there is a lot of git/github howto, you could take this one is for reference.

**Pull request closed without merging**